cli/src/alluxio.org/cli/cmd/job/load.go

```diff
@@ -12,46 +12,46 @@
 package job
 
 import (
```

Comment:
@JiamingMai @elega
please don't approve or merge PRs that reformat the entire file unless the purpose of the PR is to format. in addition to causing the back and forth formatting changes that will likely happen in the future, it also breaks git history tracking of who modified which line